packages/directory/addon/components/dir-item-name-cell.js

```diff
@@ -33,6 +33,15 @@ export default Component.extend({
     return `${type}.${singularType}`;
   }),
 
+  /**
+   * @property {String} itemId - the id of the model or the tempId of an unsaved model
+   */
+  itemId: computed('value', function() {
+    let model = get(this, 'value');
+
+    return get(model, 'tempId') || get(model, 'id');
```

Comment:
It should be the other way. `id` if present, otherwise `tempId`.

Reply:
It shouldn't matter since `tempId` will be `null` if `id` is present.
https://github.com/yahoo/navi/blob/e1fdfa433f0d1038fc51345c18f7524ad04d3fc7/packages/core/addon/models/report.js#L38